lib/menu.rb

```diff
@@ -0,0 +1,26 @@
+class Menu
```

Comment:
Nice simple code, but I think Menu currently has two responsibilities:
1) maintaining a list of menu items
2) Pretty printing the menu for a user